distributed/tests/test_scheduler.py

```diff
@@ -2168,7 +2169,7 @@ async def test_unknown_task_duration_config(s, a, b):
     assert s.idle_since == s.time_started
 
 
-@gen_cluster(client=True, timeout=1000)
+@gen_cluster(client=True, timeout=None)
```

Comment:
This seems off.  `timeout=None` can cause CI to hang without giving us much feedback.

Reply:
pytest-timeout will kill everything after 300 seconds. You'll never reach the 1000 seconds listed there.